qulice-checkstyle/src/test/java/com/qulice/checkstyle/ChecksTest.java

```diff
@@ -78,6 +78,7 @@
         "ProtectedMethodInFinalClassCheck",
         "NonStaticMethodCheck",
         "ConstantUsageCheck",
+        "LocalFinalVariableNameCheck",
```

Comment:
@jrdalpra we use `ChecksTest` only for Qulice custom checks. Please add test cases only in `CheckstyleValidatorTest`.